build.yaml

```diff
@@ -90,24 +90,52 @@ common_cli:
     #
     export venv=/tmp/v
 
-    os() {
-      # Determine OS platform
-      source /etc/os-release
-      echo \$ID
+    function os {
+      # Let's figure out if this is OSX or Linux
+      # and return the OS type (or the distribution for a Linux OS)
+      _nix_flavour=\$(uname)
+      if [ 'Darwin' == "\${_nix_flavour}" ]; then
+        echo "OSX"
+      else
+        # Determine OS platform
+        source /etc/os-release
+        echo \${ID}
+      fi
     }
+    function is_ubuntu {
```

Comment:
I'm curious why these functions don't use the exit code. Then they could be used like
```sh
if is_ubuntu; then
    ...
fi
```